x-pack/plugins/security_solution/public/notes/store/notes.slice.ts

```diff
@@ -58,7 +58,7 @@ export interface NotesState extends EntityState<Note> {
     direction: 'asc' | 'desc';
   };
   filter: string;
-  userFilter: string;
```

Comment:
Just a quick check, but none of the state in this file is persisted in local storage right? Just making sure we don't have to migrate any existing UI state for users on upgrade...

Reply:
yup, it's all in Redux only

Reply:
plus this code is new in `8.16` so it has not been deployed yet and also still behind a feature flag disabled by default